qasp/agile_bpa_qasp.md

```diff
@@ -0,0 +1,115 @@
+assurance to ensure call order standards are achieved. The contractor
+shall perform all work required in a satisfactory manner in accordance
+with the requirements of the PWS. The COR/ACOR shall notify the CO for
+appropriate action if it is likely that the contractor will not achieve
+successful final delivery of the software code in accordance with the
+performance objectives and acceptable quality levels (AQLs) identified
+below.
+
+PERFORMANCE REQUIREMENTS MATRIX
+-------------------------------
+
+The COR/ACOR will evaluate the performance objectives through
+surveillance as reflected below by reviews and acceptance of work
+products and services. As indicated, the COR/ACOR will assess progress
+towards the final delivered software code. Note that the performance
+requirements listed below are required for the final deliverable.
+However, the sprints and incremental delivery of code will be assessed
+by the Government to ensure that the contractor is on a path to
+successful final delivery.
+
+Deliverable or Required Services Performance Standard(s):
+
+  **Deliverable or Required Services** | **Performance Standard(s)** | **Acceptable Quality Level (AQL)** | **Method of Surveillance**
+   ----------- | ---------------- | -------------- | ---------------
+   Tested Code | Code delivered under the order must have substantial test code coverage and a clean code base. | Minimum of 90% test coverage of all relevant code | Combination of manual review and automated testing, using agreed-upon publicly-available SaaS products
+   Accessible | Client-side rendering must conform with section 508 standards. | 0 errors reported for 508 Standards using an automated scanner and 0 errors reported in manual testing |  http://squizlabs.github.io/HTML_CodeSniffer/ or https://github.com/pa11y/pa11y
+   Deployed | Code must successfully build and deploy into staging environment. | Successful build with a single command | Combination of manual review and automatic testing
+   Documented | All dependencies (and licenses for dependencies) are listed and all major functions are documented. | All dependencies are listed and the licenses are documented. Software/source code is documented. System diagram is provided. | Combination of manual review and automatic testing
+   Available | Code must be stored in a version-controlled open-source repository. | All of the code needed to run the front end of the prototype must be available. | 18F will assess code availability.
+   User research | Usability testing and other user research methods must be conducted at regular intervals throughout the development process (not just at the beginning or end). |  Artifacts from usability testing and/or other research methods with end-users are available at the end of every applicable sprint, in accordance with the vendor’s research plan. | 18F will evaluate the artifacts based on a research plan provided by the vendor at the end of the second sprint.
```

Comment:
This implies that a research plan is a deliverable, which I think we are going back and forth on and have leaned on it not requiring a formal deliverable.